src/components/Basket.vue

```diff
@@ -5,17 +5,17 @@
     </h1>
     <div v-else>
       <h2 class="dis">
-        You have {{ $store.state.shoppingCart.userPoints }} ponts for shopping
+        You have {{ $store.getters.displayUserPoints }} points for shopping
```

Comment:
To tak nie może być, nie powinno korzystać się z zmiennej globalnej **$store**
Ogarnij sobie to tak

```js
import { mapGetters } from 'vuex';

export default {
 // ...
 computed: mapGetters([
    'getUserPoints',
  )]
}
```

Też zamień **displayUserPoints** na te get. Bo to wygląda jak akcja / mutacja